ts/features/bonus/bpd/screens/details/transaction/BpdEmptyTransactionsList.tsx

```diff
@@ -38,9 +30,7 @@ const BpdEmptyTransactionsList: React.FunctionComponent = () => (
       </H4>
     </InfoBox>
     <View spacer={true} large={true} />
-    <Markdown cssStyle={CSS_STYLE}>
-      {I18n.t("bonus.bpd.details.transaction.detail.summary.bottomSheet.body")}
-    </Markdown>
+    {bottomSheetBpdTransactionsBody}
```

Comment:
```suggestion
    {bottomSheetBpdTransactionsBody()}
```

For the same reason as the previous comment.